src/components/ComputeModal.js

```diff
@@ -1095,14 +1101,45 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       }),
       div({ style: { lineHeight: 1.5 } }, [
         p([
-          `This cloud environment will be created in the region ${computeConfig.computeRegion.toLowerCase()}. `,
-          `Copying data from your workspace bucket in ${bucketLocation.toLowerCase()} may incur network egress charges.`
+          'This cloud environment will be created in the region ',
+          strong(`${computeConfig.computeRegion.toLowerCase()}.`),
+          ' Copying data from your workspace bucket in ',
+          strong(`${bucketLocation.toLowerCase()}`),
+          ' may incur network egress charges.'
+        ]),
+        h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/360058964552', ...Utils.newTabLinkProps }, [
+          'For more information please read the documentation.',
+          icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
         ]),
+        p(['If you want your VM in ',
+          strong(`${computeConfig.computeRegion.toLowerCase()}`),
+          ' continue. Otherwise, go back to select another location.'])
```

Comment:
Should there be a comma before `continue`?

```suggestion
          ', continue. Otherwise, go back to select another location.'])
```

Same for [line 1140](https://github.com/DataBiosphere/terra-ui/pull/2680/files#diff-149d7ad595448fb2f6bedcff6dde5f4cbcccd8d5148fed8820ce88190a3bc38bR1142).

Reply:
Yes, fixed.